public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -341,27 +349,7 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
       })),
     };
 
-    return queryFn(requestParams).pipe(
-      map((response) => response.data),
-      catchError((err: FetchError) => {
```

Comment:
I'm a little wary removing this, but I actually couldn't see what good it was doing and if it is still being run in the first place. Let me know if you think I'm missing something, but I suspect this is just legacy cruft that doesn't do much at this point